src/Connections/CommandConnection.ts

```diff
@@ -31,6 +31,10 @@ export abstract class CommandConnection<StateType extends IConnectionState = ICo
         return (this.state.commandPrefix || this.defaultCommandPrefix) + " ";
     }
 
+    public conflictsWithCommandPrefix(commandPrefix: string) {
+        return this.commandPrefix == commandPrefix + " ";
```

Comment:
Should we `toLower()` both of these to check for casing clashes?

Reply:
IMO no, because command prefixes themselves are case-sensitive.